scripts/test.sh

```diff
@@ -81,26 +102,118 @@ start_packager() {
 }
 
 xctest() {
-  local dest="$(xcrun simctl list devices | grep -v unavailable | grep -m 1 -o '[0-9A-F\-]\{36\}')"
+  # note: the DEVELOPER_DIR and IOS_SIM_DEVICE sections sould be kept in sync with setup_ios_simulator
```

Comment:
Because the Jenkins and non-Jenkins (read: developer workstation) routes are different, they both need to have the DEVELOPER_DIR and IOS_SIM_DEVICE variables setting code. That should stay in sync to keep the behaviors the same.